### ISSUE TITLE
Can only sync one wallet at a time

### ISSUE THREAD

**Issue by Transisto** [romanz/electrs#566]
When one wallet is being synced all other connection stop completely.

I'm using the default setting on MyNode, Is there something that can be done to enable simultaneous connections?

Is electrs multi threaded too? Doesn't seems like it's using more than 50% of a rPi CPU when at work.

**Comment by Kixunil** [comment]
`electrs` is multithreaded etc but it uses a single connection to bitcoind. So if the connection is exhausted by one thread syncing then the other needs to wait. I'm not sure if there's anything we can do about it, having many connections to bitcoind would mess with it's management of peers.

**Comment by Kixunil** [comment]
Yes it is, if those friends use it at the same time. I'd like this fixed too. I will try to figure out something.
